diff --git a/build/server/redirect-back.js b/build/server/redirect-back.js
index 5e2029703852c8ad6002112a10b63c3a963c0960..4ee07708a9f2fd7d01db82d72d1cb7f7191005d7 100644
--- a/build/server/redirect-back.js
+++ b/build/server/redirect-back.js
@@ -1,4 +1,4 @@
-import { UNSAFE_DataWithResponseInit, data } from "react-router";
+import { data } from "react-router";
 /**
  * Create a new Response with a redirect set to the URL the user was before.
  * It uses the Referer header to detect the previous URL. It asks for a fallback
